src/content/docs/opentelemetry/best-practices/opentelemetry-best-practices-resources.mdx

```diff
@@ -128,6 +129,26 @@ Relationships between a service and host entity require that the service
 includes the `host.id` resource attribute and that it matches the `host.id` of
 the host it is running on.
 
+### Service to AWS resources 
+
+#### Service to AWS Elasticache Memcached Cluster 
+
+Relationships between a service and AWS Memcached Cluster entity requires the following:
```

Comment:
Let's link the word "service" to the [services](https://docs.newrelic.com/docs/opentelemetry/best-practices/opentelemetry-best-practices-resources/#services) section.